h/templates/activity/search.html.jinja2

```diff
@@ -251,7 +251,19 @@
       </form>
     </section>
 
-    {{ panel('group_invite', group.url) }}
+    <h3 class="group-invite__title">
+      {% trans %}Invite new members{% endtrans %}
+    </h3>
+    Sharing the link below lets people join this group:
+    <div class="group-invite__container js-copy-button">
+      <input class="group-invite__input js-select-onfocus"
+             data-ref="input"
+             value="{{ group.url }}">
+      <button class="group-invite__clipboard-button"
+              data-ref="button"
+              title="Copy to clipboard">
+      {{ svg_icon('copy_to_clipboard') }}
```

Comment:
Ah, I see this problem was present in the place where this was moved from. Still, now is a good time to fix it.